apps/src/code-studio/components/Libraries/LibraryCreationDialog.jsx

```diff
@@ -71,9 +71,16 @@ class LibraryCreationDialog extends React.Component {
   };
 
   publish = () => {
+    let formElements = document.getElementById('selectFunction').elements;
+    let selectedFunctionList = [];
+    [...formElements].forEach(element => {
+      if (element.type === 'checkbox' && element.checked) {
+        selectedFunctionList.push(this.state.sourceFunctionList[element.value]);
+      }
+    });
     let libraryJson = libraryParser.createLibraryJson(
       this.state.librarySource,
-      this.state.selectedFunctionList,
+      selectedFunctionList,
       this.state.libraryName
```

Comment:
Were you planning to add libraryDescription here?

Reply:
Yes, still in progress.